codewars22.py

```diff
@@ -0,0 +1,18 @@
+# https://www.codewars.com/kata/53f40dff5f9d31b813000774/train/python
+
+def generate_hashtag(s):
+    if s == "" or len(s) >= 141:
+        return False
+    else:
+        s_list = []
+        start = 0
+        for index, char in enumerate(s):
+            if char == " ":
+                s_list.append(s[start:index])
+                start = index + 1
```

Comment:
Congratulations for creating the working solution! Vast majority of the times I say that you shouldn't worry about how it works, but that it works overall. However, this time I truly believe you could do it a bit more efficient, because I've seen your possibilities with another Katas. In your solution, I think there's a few steps to many. 

Hint: Instead of declaring an empty `s_list`, declare an empty (or already containing a hashtag) string. In the loop you can append the extracted words not to the list but to the aforementioned string with simple `+=` that you know very well already. You can capitalize the words in the same line, where you append them. Good luck!

Reply:
Perfect! Works faster and looks cleaner as well! Thank you very much, merging it to main branch.